ontology/uco-bfo.ttl

```diff
@@ -66,10 +66,18 @@ uco-core:Event
 		;
 	.
 
+uco-core:Facet
+	rdfs:subClassOf obo:BFO_0000020 ;
+	.
+
```

Comment:
`obo:BFO_0000020` - [specifically dependent continuant](https://github.com/BFO-ontology/BFO-2020/blob/5ed468765ea5655a0ff9a93877fd059f294a152d/src/owl/bfo-core.ttl#L860)

Reply:
On review of the [definition of "realizable entity,"](https://github.com/BFO-ontology/BFO-2020/blob/5ed468765ea5655a0ff9a93877fd059f294a152d/src/owl/bfo-core.ttl#L830), I think specifically dependent continuant is the right alignment.  `Facet` under gUFO fits _under_ gUFO's analogous class to BFO's specifically dependent continuant, but BFO doesn't happen to have an analogous class.